components/AccountOverview/index.tsx

```diff
@@ -175,7 +219,10 @@ const AccountOverview: React.FC<AccountOverviewProps> = ({
           deployer={deployerAddr}
           deployTxHash={account.smart_contract?.deployment_tx_hash}
           udt={account.udt}
-          isVerified={!!account.smart_contract?.abi}
+          address={account.eth_address}
+          isVerified={verifyStatus === 'perfect'}
+          isSubmitted={!!account.smart_contract?.deployment_tx_hash}
```

Comment:
`account.smart_contract.deployment_tx_hash` will be always `true` if it's a contract account because `deployment_tx_hash` is determined whether the contract is verified.